### DISCUSSION TITLE
Welcome to symbax Discussions!

### DISCUSSION THREAD

**Discussion by SymbaX** [SymbaX/symbax#38] [Announcements]
## 👋 ようこそ!
   私たちはコミュニティの他のメンバーとつながる場所としてディスカッションを使用しています。 次のことを願っています。
   ※気になることは質問してください。
   * アイデアを共有します。
   * 他のコミュニティメンバーと交流しましょう。
   *他人を歓迎し、オープンマインドです。 ここは私たちのコミュニティであることを忘れないでください
   一緒に築きましょう💪。

   まず、以下にコメントして自己紹介と、このコミュニティでの活動について教えてください。

**Comment by ENDOTAKUMI** [comment]
ディスカッション機能有効にしたー

**Reply by MACHIDAMinoru** [reply]
最&高✌️
